devtools/cli/distribution/jreleaser.yml

```diff
@@ -0,0 +1,47 @@
+project:
+  name: quarkus
+  description: CLI for Quarkus
+  longDescription: |
+    Create projects, manage extensions and perform build and development tasks for Quarkus,
+    the Kubernetes Native Java stack tailored for OpenJDK HotSpot and GraalVM.
+  website: https://quarkus.io
+  authors:
+    - Quarkus Community
+  license: Apache-2
+  java:
+    groupId: io.quarkus
+    artifactId: quarkus-cli
+    version : 11
+  tags:
+    - cli
+    - quarkus
+    - java
+
+release:
+  github:
+    owner: quarkusio
+    name: quarkus
+    tagName: "{{projectVersion}}"
+    overwrite: false
+    update: true
+    skipTag: true
+    changelog:
+      enabled: false
+    milestone:
+      close: false
+
+distributions:
+  quarkus:
+    type: SINGLE_JAR
+    java:
+      mainClass: io.quarkus.cli.QuarkusCli
+    brew:
+      active: release
+      extraProperties:
+        javaSkip: true
+    artifacts:
+      - path: target/quarkus-cli-{{projectVersion}}-runner.jar
+
+
+announce:
+  enabled: false
```

Comment:
If no announcers will be used then you can completely skip this block, it's disabled by default.

Reply:
y.. reading docs wasn't sure, I went for explicitness, which has the bonus of reminding me that it is there. ;)

Reply:
Understood. You can always verify settings by invoking `jreleaser config`. You also fully resolved model at `out/jreleaser/trace.log` every time you invoke a command (except for `init` & `template`) so that you can tell what settings JReleaser used.